communities/src/lib.rs

```diff
@@ -71,6 +73,18 @@ pub mod pallet {
 		type TrustableForNonDestructiveAction: EnsureOrigin<Self::RuntimeOrigin>;
 
 		type WeightInfo: WeightInfo;
+
+		#[pallet::constant]
+		type MaxCommunityIdentifiers: Get<u32>;
+
+		#[pallet::constant]
+		type MaxCommunityIdentifiersPerGeohash: Get<u32>;
+
+		#[pallet::constant]
+		type MaxLocationsPerGeohash: Get<u32>;
+
+		#[pallet::constant]
+		type MaxBootstrappers: Get<u32>;
```

Comment:
Do we need to have them configurable, or is this just a design choice? I am not against it, just curious. :)

Reply:
similar to [here](https://github.com/paritytech/substrate/blob/6fa7fe1326ecaab9921c2c3888530ad679cfbb87/frame/democracy/src/lib.rs#L289)